package.json

```diff
@@ -4,7 +4,7 @@
   "description": "hapi plugin for @sentry/node",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "ava test.js --serial",
```

Comment:
Sentry's doing some complicated things to help unify the browser/node code now. I didn't read enough to 100% understand it, but a "hub" depends on `global.__SENTRY__`:

https://github.com/getsentry/sentry-javascript/blob/81a34127cde1cbf97e634c3c914a22e03d46a952/packages/node/test/index.test.ts#L19

Short of a way to prevent that global access (e.g. to supply our own context) you have to run tests serially otherwise all the `Sentry.init` calls stomp over each other. You'll definitely see that if you remove `--serial` the tests fail. The `beforeSend` settings overwrite each other and so all but one test that depends on that times out.

Reply:
I tried it without `--serial` and got the described errors. So this seems the only way out.

Reply:
It's really this that's the problem:

https://github.com/getsentry/sentry-javascript/blob/81a34127cde1cbf97e634c3c914a22e03d46a952/packages/core/src/sdk.ts#L23-L25

It does seem that domains _might_ be a workaround:

https://github.com/getsentry/sentry-javascript/blob/81a34127cde1cbf97e634c3c914a22e03d46a952/packages/hub/src/hub.ts#L327-L357

But given limited opportunity for actual parallelization in these tests it doesn't seem worth it.